tx_origin_authorisation_misuse/README.md

```diff
@@ -0,0 +1,53 @@
+# tx.origin misuse
+A Victim contract identifies caller by tx.origin instead of msg.sender 
+That may allow some external malicious smart contract to act on behalf of origin user and attack Victim contract
+
+## Attack Scenario
+
+Some developer (not necesarily attacker, neither victim)  creates a service FreeSuperSecuredWalletsFactory (Let's call him Danny) 
```

Comment:
Can you try to focus more on technical details? For example it doesn't matter that:
* the service has been created by some developer
* that the developer's name is Danny as he's not mentioned anywhere later on

I.e. the example assumptions should probably be either  rephrased a bit or moved to previous section.

Reply:
Also we should format it better, https://guides.github.com/features/mastering-markdown/ along with reviewing other files in this repo might help you with that (there's also a `preview` button to see how your PR would be rendered: [example](https://github.com/trailofbits/not-so-smart-contracts/blob/1913579fe6f4c01fa35b099a9f6898663eea2f71/tx_origin_authorisation_misuse/README.md)).

For example we might introduce bullet points to some of the places and make contract names or functions rendered as inline code like this: `FreeSuperSecuredWalletsFactory`.

Reply:
take a look if formating is ok now